File: HappyBirthday/app/src/main/java/com/example/happybirthday/Splash.java
package com.example.happybirthday;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import com.example.happybirthday.MainActivity;
import com.example.happybirthday.R;

public class Splash extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent i = new Intent(Splash.this, MainActivity.class);
                startActivity(i);
                finish();
            }
        }, 3000);
    }
}

File: Dice_Roller/app/src/main/java/com/example/dice_roller/Splash.kt
package com.example.dice_roller

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import com.example.dice_roller.MainActivity
import com.example.dice_roller.R

class Splash : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        Handler().postDelayed({

            intent = Intent(this, MainActivity::class.java)
            startActivity(intent)
            finish()
        },3000)


    }
}